targets/f18/api_symbols.csv

```diff
@@ -13,6 +13,7 @@ Header,+,applications/services/gui/icon_i.h,,
 Header,+,applications/services/gui/modules/button_menu.h,,
```

Comment:
Bump API version minor

Reply:
Please double check if I'm doing this correctly